[
    {
        "user": "U01N3ET8N82",
        "type": "message",
        "ts": "1731627404.930209",
        "client_msg_id": "dc138d1b-6c62-4eb7-85cf-1b8dcfea5931",
        "text": "ah sorry for missing all this. we are in the midst of moving and my phone with slack installed broke. first time i've logged in in some time",
        "team": "T01DW9DLB2A",
        "user_team": "T01DW9DLB2A",
        "source_team": "T01DW9DLB2A",
        "user_profile": {
            "avatar_hash": "6c5b09058425",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-07-26\/7489004466548_6c5b09058425ecbe2d89_72.jpg",
            "first_name": "Michael",
            "real_name": "Michael Chirico",
            "display_name": "Michael Chirico (he\/him)",
            "team": "T01DW9DLB2A",
            "name": "michaelchirico4",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "thread_ts": "1730119641.271779",
        "parent_user_id": "U027JEHGZRA",
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "Ys3mk",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "ah sorry for missing all this. we are in the midst of moving and my phone with slack installed broke. first time i've logged in in some time"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U01N3ET8N82",
        "type": "message",
        "ts": "1731627452.740939",
        "client_msg_id": "bb55f55e-1a27-4477-9b92-a40a1914b799",
        "text": "I recall some issue with deconflict_add_continue.py... it wasn't in the expected place but hadn't disappeared... i wound up copy-pasting it from gist last time I think?",
        "team": "T01DW9DLB2A",
        "user_team": "T01DW9DLB2A",
        "source_team": "T01DW9DLB2A",
        "user_profile": {
            "avatar_hash": "6c5b09058425",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-07-26\/7489004466548_6c5b09058425ecbe2d89_72.jpg",
            "first_name": "Michael",
            "real_name": "Michael Chirico",
            "display_name": "Michael Chirico (he\/him)",
            "team": "T01DW9DLB2A",
            "name": "michaelchirico4",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "thread_ts": "1730119641.271779",
        "parent_user_id": "U027JEHGZRA",
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "MZcWf",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "I recall some issue with deconflict_add_continue.py... it wasn't in the expected place but hadn't disappeared... i wound up copy-pasting it from gist last time I think?"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U01N3ET8N82",
        "type": "message",
        "ts": "1731627494.303759",
        "client_msg_id": "69b6fd62-f50a-4bf8-9cf1-bd76e6a90453",
        "text": "thank you for handling <@U027JEHGZRA>!!!",
        "team": "T01DW9DLB2A",
        "user_team": "T01DW9DLB2A",
        "source_team": "T01DW9DLB2A",
        "user_profile": {
            "avatar_hash": "6c5b09058425",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-07-26\/7489004466548_6c5b09058425ecbe2d89_72.jpg",
            "first_name": "Michael",
            "real_name": "Michael Chirico",
            "display_name": "Michael Chirico (he\/him)",
            "team": "T01DW9DLB2A",
            "name": "michaelchirico4",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "thread_ts": "1730119641.271779",
        "parent_user_id": "U027JEHGZRA",
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "4j1y5",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "thank you for handling "
                            },
                            {
                                "type": "user",
                                "user_id": "U027JEHGZRA"
                            },
                            {
                                "type": "text",
                                "text": "!!!"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U027JEHGZRA",
        "type": "message",
        "ts": "1731627606.278169",
        "edited": {
            "user": "U027JEHGZRA",
            "ts": "1731627631.000000"
        },
        "client_msg_id": "12a5888a-555d-4536-af10-c72941bffa73",
        "text": "re deconflict.py: it's not available at the beginning of the process of resolving the git rebase conflicts .. as it's not commited there \"yet\" :) it pops up after resolving s few hundred commits .. and the multi-file updates etc even later. in short, the repo is not the best place for it, but manageable problem",
        "team": "T01DW9DLB2A",
        "user_team": "T01DW9DLB2A",
        "source_team": "T01DW9DLB2A",
        "user_profile": {
            "avatar_hash": "67eb97d51d7a",
            "image_72": "https:\/\/avatars.slack-edge.com\/2021-07-07\/2251315439203_67eb97d51d7a584014d2_72.jpg",
            "first_name": "Gergely",
            "real_name": "Gergely Daróczi",
            "display_name": "Gergely Daróczi",
            "team": "T01DW9DLB2A",
            "name": "daroczig",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "thread_ts": "1730119641.271779",
        "parent_user_id": "U027JEHGZRA",
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "qcuow",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "re deconflict.py: it's not available at the beginning of the process of resolving the git rebase conflicts .. as it's not commited there \"yet\" :) it pops up after resolving s few hundred commits .. and the multi-file updates etc even later. in short, the repo is not the best place for it, but manageable problem"
                            }
                        ]
                    }
                ]
            }
        ]
    }
]